Main.sublime-menu

```diff
@@ -156,6 +156,11 @@
         "id": "lsp",
         "caption": "-"
       },
+      {
+        "caption": "LSP: Show Code Lens",
```

Comment:
I would use "Lenses" here (plural, like the other items), and probably for the command name too.

Maybe we should add the checkbox for the inlay hints toggle as well?